abis/linux/limits.h

```diff
@@ -2,5 +2,7 @@
 #define _ABIBITS_LIMITS_H
 
 #define IOV_MAX 1024
+#define NAME_MAX 255
```

Comment:
The commit message / PR name is wrong, since it adds `NAME_MAX` too. Though we already define `NAME_MAX` in `ansi/limits.h`, so we shouldn't define it here too. Perhaps that + a few of the other constants there should be moved here.

Reply:
fwiw, name_max is the system-specific variant. See also: https://github.com/ArsenArsen/mlibc/commit/6d62dbce05ffb6353611a027650784c10f6c8253